website/pages/en/support.js

```diff
@@ -16,8 +16,8 @@ The React Native for Windows (RNW) Team strives to provide full support for the
 | Version | Support Phase | Release Date | Active Support Start | Maintenance Support Start | End of Support |
 | -- | -- | -- | -- | -- | -- |
 | [main](https://www.npmjs.com/package/react-native-windows/v/canary) | [Canary](#canary-support) | *N/A* | *N/A* | *N/A* | *N/A* |
-| [0.73](https://www.npmjs.com/package/react-native-windows/v/preview) | [Preview](#preview-support) | *TBD* | *TBD* | *TBD* | *TBD* |
-| [0.72](https://www.npmjs.com/package/react-native-windows/v/latest) | [Active](#active-support) | 06/23/2023 | 06/23/2023 | *TBD* | *TBD* |
+| [0.73](https://www.npmjs.com/package/react-native-windows/v/latest) | [Active](#active-support) | 12/11/23 | 12/11/23 | *TBD* | *TBD* |
+| [0.72](https://www.npmjs.com/package/react-native-windows/v/v0.72-stable) | [Active](#active-support) | 06/23/2023 | 06/23/2023 | 01/11/23 | 03/11/23 |
```

Comment:
should be the last day of the next month, also the correct year
```suggestion
| [0.72](https://www.npmjs.com/package/react-native-windows/v/v0.72-stable) | [Active](#active-support) | 06/23/2023 | 06/23/2023 | 01/31/24 | 03/30/24 |
```